docs/content/gs-adding-users-proc.adoc

```diff
@@ -1,171 +1,188 @@
 [id='gs-adding-users-proc']
 
-= User Management
+ifdef::env-github[]
+:imagesdir: ../images/
+endif::[]
 
-This section covers how to manage users within the Integreatly environment.
+= Managing Users
 
-== Accessing the SSO Admin Console
-- Login to the OpenShift console as the `customer-admin` user (The credentials for this user were sent with your welcome email).
-- Navigate to the `sso` project.
-- Go to *Applications* > *Routes*
-- Click on the `sso` route hostname to open the Single Sign-On (SSO) landing screen.
+How to manage users within the {PRODUCT} environment.
 
-image:../assets/images/gs-adding-users-sso-route.png[SSO route]
+== Accessing the Red Hat Single Sign-On administration console
 
-- Click on the `Administration Console` link on the SSO landing screen.
-- Login to the SSO administration console as the `customer-admin` user.
+To access the administration console navigate to the following URL, replacing `<cluster-host>` with the hostname of your cluster:
 
-== Viewing Users in the SSO Admin Console
-- Navigate to the *Users* menu.
-- Click the *View all users* button at the top of the page. This will show the list of users currently available in the Integreatly environment.
+----
+https://<cluster-host>/auth/admin/openshift/console
```

Comment:
I think this should be `https://sso-sso.apps.<cluster-id>.openshiftworkshop.com/auth/realms/openshift`. Tried it with cluster host as `master.<cluster-id>.openshiftworkshop.com` and that doesn't work. I think the `sso-sso.apps` needs to be specified

Reply:
I added `sso-sso.apps.` don't think we can mention openshiftworkshop.com, right?

Reply:
Good catch. I don't think so, it might be different for OSD. @maleck13

Reply:
punted to https://issues.jboss.org/browse/INTLY-1167